src/k2/module/k23si/client/k23si_client.h

```diff
@@ -99,7 +99,8 @@ k2::TSO_ClientLib& _tsoClient;
     seastar::future<Status> makeCollection(const String& collection, std::vector<k2::String>&& rangeEnds=std::vector<k2::String>());
     seastar::future<K2TxnHandle> beginTxn(const K2TxnOptions& options);
     static constexpr int64_t ANY_VERSION = -1;
-    seastar::future<Schema> getSchema(const String& collectionName, const String& schemaName, int64_t schemaVersion);
+    seastar::future<seastar::lw_shared_ptr<dto::Schema>> getSchema(const String& collectionName, const String& schemaName, int64_t schemaVersion, bool doCPORefresh=true);
+    seastar::future<Status> refreshSchemaCache(const String& collectionName);
```

Comment:
It's a little confusing to have this as public. Am I supposed to use it as a user of the client lib?